### PR TITLE
015/master

### PR DESCRIPTION
**Pull request by neugierige**
addressing 2 things from issue #17: 1) type in the instructions, and 2) clarify Checkpoint 1 and update code to make it run as expected

### CODE REVIEW

File: study.md [hunk 1]
```diff
@@ -112,26 +112,27 @@ var myModule = (function(arg, transform) {
 1.  Leave `f` as-is or alter it to create a more interesting module.
 1.  Leave the statements toward the end of the document as-is or alter them to
     do something more interesting.
-1.  Run the script with `node index.js`.
+1.  Run the script with `node inline.js`.
 
 ### Checkpoint 1
 
 1.  Open [`lib/inline.js`](lib/inline.js).
 1.  Put a pair of parens (`(` and `)`) around our declaration of `f`.
```
Comment (gaand) [inline study.md]: Maybe say:

```diff
-1.  Put a pair of parens (`(` and `)`) around our declaration of `f`.
+1.  Put a pair of parens (`(` and `)`) around the function assigned to `f`.

File: study.md [hunk 2]
```diff
@@ -112,26 +112,27 @@ var myModule = (function(arg, transform) {
...
+    Assign this function to a new variable, and update the assignment of
+    `ourModule` to equal this new variable.
```
Comment (gaand) [inline study.md]: This addition seems unnecessary so I suggest removing it.

**Review by gaand — COMMENTED** [review]
A couple of notes.

File: study.md (hunk 2 above)
Comment (neugierige) [reply study.md]: Agreed. That was my original proposed change, I didn't intend to include it in the commit.

**Pull request merged**